server/test/letra.test.js

```diff
@@ -0,0 +1,28 @@
+const request = require('supertest');
+const app = require('../app');
+const languages = require(`../data/languages`);
+
+describe('/', () => {
+  it('returns a 200 response', async () => {
+    const res = await request(app).get('/');
+    expect(res.statusCode).toEqual(200);
+  });
+  it('returns a message', async () => {
+    const expectedMessage = 'Hi There! Nothing much here, just a server for the letra extension!';
+    const res = await request(app).get('/');
+    expect(res.text).toEqual(expectedMessage);
+  });
+});
+
+describe('/languages', () => {
+    it('returns a 200 response', async () => {
+      const res = await request(app).get('/languages');
+      expect(res.statusCode).toEqual(200);
+    });
+    it('returns languages as the response body', async () => {
+      const res = await request(app).get('/languages');
+      console.log(res);
```

Comment:
Is there a reason for the `console.log` here? :)

Reply:
Good catch. Looks like I had forgotten to remove those earlier. :sweat_smile: They should be removed now.

Reply:
Thank you!!